### ISSUE TITLE
Error 0x80101104 

### ISSUE THREAD

**Issue by Alpitz** [SonicMastr/godot-vita#54]
I keep trying to download my game on vita but it says Error 0x80101104 
what i can do

**Comment by shortbow123** [comment]
I have worked on this problem myself for 8 months. From what I have found it is an issue with how the project is exported and packaged into the vpk. I fixed this by changing the version to 0.01. I think the script for building the vpk needs to be reworked, I just aint bright enough to do it.

<img width="387" alt="Screenshot 2024-10-16 at 4 06 50 AM" src="https://github.com/user-attachments/assets/9b187303-d8d2-464e-b0dd-2ac1d87a78d4">
Compiled on M3 MacBook macOS 15.0 last published build of Godot-vita

TLDR/SEO:
Error 0x80101104 on the vita means you need to set version in export settings to 0.01
